client/src/views/files/FoldersPage.vue

```diff
@@ -207,59 +207,121 @@ import { routerNavigateTo } from '@/router';
 import { FileAction } from '@/views/files/FileContextMenu.vue';
 import MsActionBar from '@/components/core/ms-action-bar/MsActionBar.vue';
 import FileUploadModal from '@/views/files/FileUploadModal.vue';
-
+import { NotificationCenter, Notification, NotificationKey, NotificationLevel } from '@/services/notificationCenter';
+import * as parsec from '@/parsec';
+import { join as pathJoin } from '@/common/path';
+import { useI18n } from 'vue-i18n';
+import { getText } from '@/components/core/ms-modal/MsTextInputModal.vue';
```

Comment:
The name is really not clear.
Should atleast ne something like `getTextInputModal` or `showTextInputModal`

Reply:
The function does not return a text input modal, and while it does show a text input modal, its role is to get a string from the user. I do agree that `getText` is kinda meh but I didn't really find something better. `getTextFromUser` maybe but it sounds weird too.

Reply:
On the technical side this is a createAndPresentModal function like there are others in the project.
On the UX side its role is to getTextInputfromUser.
Should be based on one or the other.

Reply:
Renamed to `getInputTextFromUser`